skip-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
@chargome 👀

Reply:
@a-hariti we should probably also rebuild if we have changes in the app directory

Reply:
yeah, I think it should be more ifelesy, ie build dev-docs otherwise build sdk docs, currently it doesn't cover every case

exploring on my personal Vercel account ...